charts/radar-jdbc-connector/values.yaml

```diff
@@ -6,7 +6,7 @@ replicaCount: 1
 
 image:
   repository: radarbase/radar-jdbc-connector
-  tag: dev
+  tag: latest
```

Comment:
@mpgxvii please change this tag to a numerical version so it would be easier to update and keep track of

Reply:
Okay sure I'll update it